package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "hamongts",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "export": "next build && next export",
+    "lint": "eslint --ignore-path .gitignore --ext js,ts,tsx src",
+    "prettier": "prettier \"**/*.+(js|jsx|ts|tsx|json|html|yml|yaml|css|md)\"",
+    "format": "yarn prettier -- --write",
+    "check-format": "yarn prettier -- --list-different",
```

Comment:
`From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded.`
Thus: remove ` -- `.